### ISSUE TITLE
google-chrome as browser

### ISSUE THREAD

**Issue by chritter** [StatCan/kubeflow-containers#33]
Hello,

It would be great to have google chrome available on the images. We need it for webscraping.

Christian

**Comment by chritter** [comment]
@brendangadd Testing software like this first in your proposed testing image environment would be useful.

**Comment by blairdrummond** [comment]
I think that Stan used headless Selenium or something for this, do you know the details of that?

**Comment by chritter** [comment]
Yes, we need `google-chrome` for selenium.

**Comment by blairdrummond** [comment]
Google-chrome specifically, or chromium or firefox? Google-chrome is not open source

**Comment by chritter** [comment]
@blairdrummond Good point. Firefox should do. Thanks!

**Comment by blairdrummond** [comment]
Can you write up a full list of requirements for this image? I will get you a customized image.

**Comment by chritter** [comment]
@blairdrummond  Thank you for supporting us and a project-related image would be great. Currently we tested it with google-chrome in a different environment.

**Comment by blairdrummond** [comment]
I'm assuming you want this in an ML-Workspaces image?

It might make sense to fork the image and go from there. Installing Chromium will be simpler than installing Google Chrome. Do you want to try that? I'm not sure if I fully understand your requirements yet

**Comment by chritter** [comment]
Yes, a ML-Workspace image would be great. We will fork the repo. Let's discuss requirements offline.
